app/controllers/setup_controller.rb

```diff
@@ -129,10 +129,12 @@ def suse_registry_mirror_params
     end
 
     if params["settings"]["suse_registry_mirror_enabled"] == "enable"
-      parameters["mirror"] = SUSE_REGISTRY_URL
+      parameters["mirror_url"] = params["settings"]["suse_registry_mirror"]["url"]
+      # not needed anymore
+      parameters.delete :url
```

Comment:
Do we really need  to remove `url` from the parameters?

Reply:
not technically